### PR TITLE
Cakebuild 1.0.0

### PR DESCRIPTION
**Pull request by stefandevo**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by stefandevo** [comment]
I was not sure how to add an icon for the step, so I added an icon.svg to the pull request as well.

**Comment by viktorbenei** [comment]
Thanks, we'll check it asap (tomorrow)!

File: steps/cakebuild/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    description: |
+      Name of the script to execute. Default is build.cake
+    is_expand: true
+    is_required: true
+    title: Script
+  script: build.cake
+- dotnetcore: "no"
+  opts:
+    description: |
+      Is the project a .NET Core project? Default is faise.
```
Comment (godrei) [inline steps/cakebuild/1.0.0/step.yml]: typo: `faise` -> `false`

File: steps/cakebuild/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+title: Execute Cake C# Script
+summary: Cake (C# Make) is a cross platform build automation system
+description: "Cake (C# Make) is a cross platform build automation system with a C#
+  DSL to do things like compiling code, \ncopy files/folders, running unit tests,
+  compress files and build NuGet packages."
+website: http://cakebuild.net/
```
Comment (godrei) [inline steps/cakebuild/1.0.0/step.yml]: `website` should point to the step's website (most of the times, it's github url), please mention the Cake's website in step's `description` instead.

**Comment by godrei** [comment]
Hi @stefandevo ,
thanks for the PR!

I attached some notes to your PR, please have a look and fix them.

Once, you finish the changes please re-share your step:
- close this pr
- delete the `cakebuild-1.0.0` branch in your steplib fork
- share the step again

**Pull request closed without merging**